website/templates/project/settings.mako

```diff
@@ -33,15 +33,13 @@
                             <li><a href="#configureAddonsAnchor">Configure Add-ons</a></li>
                         % endif
 
-                        % if include_wiki_settings:
-                            <li><a href="#configureWikiAnchor">Configure Wiki</a></li>
-                        % endif
+                        <li><a href="#configureWikiAnchor">Wiki</a></li>
 
                         % if 'admin' in user['permissions']:
-                            <li><a href="#configureCommentingAnchor">Configure Commenting</a></li>
```

Comment:
Done.  The only change I found that was needed was for Configure Comments being changed to Comments.